examples/custom_flags/custom_flags_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package custom_flags
+
+import (
+	"context"
+	"flag"
+	"os"
+	"testing"
+
+	log "k8s.io/klog/v2"
+
+	"sigs.k8s.io/e2e-framework/pkg/env"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+var (
+	test         env.Environment
+	myCustomFlag string
+)
+
+func TestMain(m *testing.M) {
+	flag.StringVar(&myCustomFlag, "my-custom-flag", "", "my custom flag for my tests")
+	// create config from flags (always in TestMain because it calls
```

Comment:
`TestMain` or `init` handler of the package. Both will do, But should be done before the `envconf.NewFromFlags()` is called. That is all. Mind updating the comment here ?

Reply:
Done. Do you also want me to update https://github.com/kubernetes-sigs/e2e-framework/blob/db4e9fe7e23d2602c34d1fe3853625e754b1fa41/examples/flags/flags_test.go#L58 since I copied from that?